### DISCUSSION TITLE
Highlight in SumatraPDF

### DISCUSSION THREAD

**Discussion by yuxuanfang** [sumatrapdfreader/sumatrapdf#4032] [Q&A] [answered]
I have the most recent pre-release version, [3.6.15941](https://www.sumatrapdfreader.org/prerelease), of SumatraPDF. I'm attempting to highlight text, and according to my online search, the suggested method is right-clicking on the selected words and clicking the scrollbar. However, the option to highlight is not appearing as expected.
<img width="188" alt="image" src="https://github.com/sumatrapdfreader/sumatrapdf/assets/147900755/11b0f726-3b0f-44d8-b98f-923f497ecad9">

**Comment by GitHubRulesOK** [comment]
Not right-click sidebar (there would be no annotation context there), you need to click on the image area in a PDF. 
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/cbe924e7-c9a1-45d1-aa6c-8e3017295c7c)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/556f2ec0-d83d-4ac9-890b-32d387da9521)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/cf4d099b-f9a3-45fc-8a77-6965c98b286d)

**Reply by yuxuanfang** [reply, marked as answer]
I think I found the answer because I opened the epub document instead of the pdf document.
